test/host/src/main/java/org/apache/teaclave/javasdk/test/host/TestEnclaveAES.java

```diff
@@ -22,13 +22,21 @@
 import org.apache.teaclave.javasdk.host.EnclaveType;
 import org.apache.teaclave.javasdk.test.common.AESSealedTest;
 import org.apache.teaclave.javasdk.test.common.AESService;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
 import java.util.Iterator;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 public class TestEnclaveAES {
+    @Before
+    public final void before() { System.out.println("enter test case: " + this.getClass().getName()); }
```

Comment:
`junit` will print these messages automatically, so we don't need to print them explicitly.

For example:

```
Running Test
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.039 sec - in Test
```